macros/nsprapiref.ejs

```diff
@@ -159,349 +159,409 @@ var text = mdn.localStringMap({
 <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference"><%=text['NSPR API Reference']%></a></li>
 <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/NSPR_build_instructions"><%=text['NSPR Build Instruction']%></a></li>
 <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Release_process"><%=text['NSPR Release Process']%></a></li>
-<li data-default-state="<%=state('Introduction to NSPR')%>"><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR"><%=text['Introduction to NSPR']%></a>
-
-<ol>
- <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#NSPR_Naming_Conventions"><%=text['NSPR Naming Conventions']%></a></li>
- <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#NSPR_Threads"><%=text['NSPR Threads']%></a>
-  <ol>
-   <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#Thread_Schedoling"><%=text['Thread Scheduling']%></a>
-    <ol>
-     <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#Setting_Thread_Priorities"><%=text['Setting Thread Priorities']%></a></li>
-     <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#Preempting_Threads"><%=text['Preempting Threads']%></a></li>
-     <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#Interrupting_Threads"><%=text['Interrupting Threads']%></a></li>
-    </ol>
-   </li>
-  </ol>
- </li>
- <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#NSPR_Thread_Synchronization"><%=text['NSPR Thread Synchronization']%></a>
-  <ol>
-   <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#Locks_and_Monitors"><%=text['Locks and Monitors']%></a></li>
-   <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#Condition_Variables"><%=text['Condition Variables']%></a></li>
-  </ol>
- </li>
- <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#NSPR_Sample_Code"><%=text['NSPR Sample Code']%></a></li>
-</ol>
+<li class="toggle">
+  <details <%=state('Introduction to NSPR')%>>
+      <summary><%=text['Introduction to NSPR']%></summary>
+      <ol>
+        <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#NSPR_Naming_Conventions"><%=text['NSPR Naming Conventions']%></a></li>
+        <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#NSPR_Threads"><%=text['NSPR Threads']%></a>
+          <ol>
+          <li><a href="/<%=locale%>/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#Thread_Schedoling"><%=text['Thread Scheduling']%></a>
```

Comment:
Existing nit: [#Thread_Scheduling](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR/Reference/Introduction_to_NSPR#Thread_Scheduling), not ``#Thread_Schedoling``. Although "Schedoling" should be a word...